tools/labs/templates/hello_assignment/sol/hello-world.c

```diff
@@ -0,0 +1,21 @@
+#include <linux/kernel.h>
+#include <linux/init.h>
+#include <linux/module.h>
+
+MODULE_DESCRIPTION("Hello World");
+MODULE_AUTHOR("Psoru Lesfo Rever");
+MODULE_LICENSE("GPL");
+
```

Comment:
Since we are here I think we should also find a better name for module author. I don't have good names now, but I think kernel people recommend to use real names here.

Reply:
Agree. It should still be a generic name. I recommend we use "Linux Kernel Labs <so2@cursuri.cs.pub.ro>", or we can create a new e-mail address.

Reply:
We should use the same author for all modules. I suggest changing this for all modules in a future commit.